dvc/dvcfile.py

```diff
@@ -200,9 +200,7 @@ def dump(self, stage, **kwargs):
         assert not isinstance(stage, PipelineStage)
         if self.verify:
             check_dvcfile_path(self.repo, self.path)
-        logger.debug(
-            "Saving information to '{file}'.".format(file=relpath(self.path))
-        )
+        logger.debug(f"Saving information to '{relpath(self.path)}'.")
```

Comment:
To address @pared 's point about lazy logging

```suggestion
        logger.debug("Saving information to '%s'", relpath(self.path))
```

Reply:
This is unnecessary, as relpath is already in string.
```pycon
>>> path = os.getcwd()
>>> %timeit logger.debug("Saving information to '%s'.", relpath(path))
69 µs ± 840 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
>>> %timeit logger.debug(f"Saving information to '{relpath(path)}'.")
66.5 µs ± 1.42 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

Reply:
@skshetry That is not for explicit convertion, but rather for lazyness. Or am I missing something and f-strings lazy here?

Reply:
you are right, f-strings are not lazy, but I don't think lazy-formatting is expensive here, it's the string conversion that is, which is done eagerly. And, this is not an expensive code path, so I'd prefer f-strings here.

This is the result that I get when the logger is disabled.
```pycon
>>> %timeit logger.debug(f"Saving information to '{relpath(path)}'.")
29 µs ± 1.03 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)

>>> %timeit logger.debug("Saving information to '%s'.", relpath(path))
29.8 µs ± 790 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

(they are very close and flip flop on which one is better.)

Reply:
right, I didn't mean that is was unbearably expensive here, just was thinking about consistency and developing a habit/rule so we always use lazy notation in logger. So could keep it as is, I'm. totally fine with this.